config.md

```diff
@@ -28,7 +28,7 @@ Changing it means creating a new derived image, instead of changing the existing
 A layer DiffID is a SHA256 digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
 Layers must be packed and unpacked reproducibly to avoid changing the layer DiffID, for example by using tar-split to save the tar headers.
 
-NOTE: the DiffID is different than the digest in the manifest list because the manifest digest is taken over the gzipped layer for `application/vnd.oci.image.layer.v1.tar+gzip` types.
+NOTE: the DiffID is different than the layer digest because the layer digest is taken over the content of the gzipped layer for `application/vnd.oci.image.layer.v1.tar+gzip` types.
```

Comment:
Your suggested language points out the DiffID being over the uncompressed content.  @coolljt0725's current wording points out the layer digest being over the (raw) gzipped content.  Both of these are important for understanding the difference, and while the uncompressed-ness of DiffID is mentioned in the previous paragraph, I think it's worth talking about both of them in this note.  I have wording for that [in my earlier suggestion][1].  Until something like #332 or #388 lands, we could just use “over the uncompressed content” instead of “over the `application/vnd.oci.image.layer.tar` content”.


[1]: https://github.com/opencontainers/image-spec/pull/328#discussion_r79444393